### ISSUE TITLE
Version 1.35.0 not working

### ISSUE THREAD

**Issue by frankdou** [dbtedman/postcss-prefixwrap#185]
Hi,

When i use latest version, got below error.
And return to 1.34.0，back to normal.

postcss.config.js

```js
module.exports = {
    plugins: [
        require("autoprefixer"), 
        require("postcss-prefixwrap")('.test'),
    ],
}
```

```
Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
Error: Cannot find module 'plugin/PostCSSPrefixWrap'
Require stack:
- /Users/.../node_modules/postcss-prefixwrap/build/plugin/PostCSS8Plugin.js
- /Users/.../node_modules/postcss-prefixwrap/build/plugin/PostCSSPlugin.js
- /Users/.../node_modules/postcss-prefixwrap/build/index.js
- /Users/.../postcss.config.js
- /Users/.../node_modules/cosmiconfig/dist/loaders.js
- /Users/.../node_modules/cosmiconfig/dist/ExplorerBase.js
```

**Comment by dbtedman** [comment]
Thanks @frankdou for reporting this issue, I am looking into a fix now.

**Comment by dbtedman** [comment]
I have made some updates and released a patch version `1.35.1` that should resolve this issue. Additionally, I am reviewing my release testing procedure to reduce the risk of this kind of issue in future.
